docs/Makefile

```diff
@@ -1,7 +1,7 @@
-DOCKER       = docker
-HUGO_VERSION = 0.50
-DOCKER_IMAGE = jojomi/hugo:$(HUGO_VERSION)
-DOCKER_RUN   = $(DOCKER) run --rm --interactive --tty --volume $(PWD):/src
+DOCKER       ?= docker
+HUGO_VERSION ?= 0.50
```

Comment:
@lmakarov I changed it back to `0.50` but believe that we should make it overridable.